### PR TITLE
Add isLatestVersion on PipelineVersion

### PR DESCRIPTION
**Pull request by qgerome**
I need this for the frontend to display easily the badge on the latest version. In the future this will become the isActiveVersion when users will be able to select a different version as the active one.

### CODE REVIEW

**Comment by cheikhgwane** [comment]
Conflicts need to be fixed.

**Comment by qgerome** [comment]
> Conflicts need to be fixed.

done

File: hexa/pipelines/schema/mutations.py [hunk 1]
```diff
@@ -1,30 +1,16 @@
...
-)
+
+from ariadne import MutationType, convert_kwargs_to_snake_case
 from django.conf import settings
 from django.core.exceptions import PermissionDenied
 from django.db import IntegrityError
 from django.http import HttpRequest
-from django.urls import reverse
-from sentry_sdk import capture_exception
+from google.api_core.exceptions import NotFound
```
Comment (cheikhgwane) [inline hexa/pipelines/schema/mutations.py:8]: I think you forgot to update this import (import NotFound exception from  hexa.files.api).

**Review by cheikhgwane — APPROVED** [review]
LGTM

**Pull request merged**